src/DIRAC/ConfigurationSystem/Agent/RucioSynchronizerAgent.py

```diff
@@ -322,26 +500,32 @@ def execute(self):
           except Duplicate:
             pass
           except Exception as err:
-            self.log.error('Cannot create identity %s for account %s : %s', dn, account, str(err))
+            self.log.error('Cannot create identity %s for account %s : %s' % (dn, account, str(err)))
```

Comment:
These logs should all separate the fixed and variable parts, e.g.

```suggestion
            self.log.error("Cannot create identity for account", "[DN: %s, account: %s. Error: %s]" % (dn, account, str(err)))
```